kubernetes/istio_details_service.go

```diff
@@ -328,6 +347,25 @@ func (in *IstioClient) GetPolicy(namespace string, policyName string) (IstioObje
 	return policy.DeepCopyIstioObject(), nil
 }
 
+func (in *IstioClient) GetMeshPolicies() ([]IstioObject, error) {
+	result, err := in.istioAuthenticationApi.Get().Resource(meshPolicies).Do().Get()
```

Comment:
Here is the same problem.
I described this in an old thread:
https://groups.google.com/forum/#!msg/kiali-dev/VKlfO6qgZZM/k0_aigJBCQAJ

Reply:
I have found out that the API response all the meshpolicies eventhough you pass the namespace as a paremeter.
For example, if you have one MeshPolicy called `default`, the following call just respond with that:
```golang
in.istioAuthenticationApi.Get().Namespace("bookinfo").Resource(meshPolicies).Do().Get()
```
So the approach I took is the one of passing one of the namespaces that the user has access to in order to retrieve this MeshPolicy. This should work in soft-multitenancy environments. The error connected to fetching all the resources will be skipped.